code/modules/surgery/organs/lungs.dm

```diff
@@ -88,7 +88,7 @@
 		return
 
 	if(!breath || (breath.total_moles() == 0))
-		if(isspaceturf(loc))
+		if(isspaceturf(H.loc))
```

Comment:
Edge case to be careful about is being in a closed locker in space.

Reply:
Yeah, this isn't perfect, to be sure---I thought about this a bit when I noticed the error, but using something like `get_turf` is going to result in a lot weird behavior, like taking in crease O2 damage when you're in a mech or a pod while on a space turf.